### PR TITLE
Added suport for respectpdb in roll_configs in scheduledRoll config.

### PR DESCRIPTION
**Pull request by sharadkesarwani**
Added suport for respectpdb in roll_configs in scheduledRoll config

# Jira Ticket
https://spotinst.atlassian.net/browse/SPOTAUT-12985

### CODE REVIEW

File: service/ocean/providers/gcp/cluster.go [hunk 1]
```diff
@@ -1420,3 +1436,54 @@ func (o *LaunchSpecShieldedInstanceConfig) SetEnableSecureBoot(v *bool) *LaunchS
 }
 
 //endregion
+
+//region cluster clusterRoll
```
Comment (chandra1-n) [inline service/ocean/providers/gcp/cluster.go:1440]: interchange the order, keep parameters first and then clusterRoll

File: examples/service/ocean/providers/gcp/create/main.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+				Maximum: spotinst.Int(1000),
+				Target:  spotinst.Int(1),
+			},
+			Compute: &gcp.Compute{
+				SubnetName: spotinst.String("default"),
+				InstanceTypes: &gcp.InstanceTypes{
+					Whitelist: []string{"e2-micro"},
+				},
+				LaunchSpecification: &gcp.LaunchSpecification{
+					ServiceAccount: spotinst.String("493916419393-compute@developer.gserviceaccount.com"),
```
Comment (chandra1-n) [inline examples/service/ocean/providers/gcp/create/main.go:63]: remove actual values from the example

**Review by chandra1-n — CHANGES_REQUESTED** [review]
please fix the comments

File: service/ocean/providers/gcp/cluster.go (hunk 1 above)
Comment (sharadkesarwani) [reply service/ocean/providers/gcp/cluster.go]: Updated.

File: examples/service/ocean/providers/gcp/create/main.go (hunk 2 above)
Comment (sharadkesarwani) [reply examples/service/ocean/providers/gcp/create/main.go]: Removed actualValue.

**Review by chandra1-n — APPROVED** [review]

**Pull request merged**